rllib/utils/torch_utils.py

```diff
@@ -172,6 +172,21 @@ def mapping(item):
     return tree.map_structure(mapping, x)
 
 
+@PublicAPI
+def copy_and_move_to_device(x: TensorStructType, device: Optional[str] = None):
```

Comment:
Can we add docstrings here to clarify what sort of copy this is?
Also add what happens to items that are not torch Tensors?
Are some of the optimizer weights not torch tensors? Because usually I'd expect this to error out if elements of the tensorstruct are not torch tensors.

Reply:
I've added a docstring, hoping it adds enough clarity :)